source/utilities/imgcat

```diff
@@ -23,13 +23,14 @@ function print_st() {
 
 function load_version() {
     if [ -z ${IMGCAT_BASE64_VERSION+x} ]; then
-        export IMGCAT_BASE64_VERSION=$(base64 --version 2>&1)
+        IMGCAT_BASE64_VERSION=$(base64 --version 2>&1)
+        export IMGCAT_BASE64_VERSION
     fi
 }
 
 function b64_encode() {
     load_version
-    if [[ "$IMGCAT_BASE64_VERSION" =~ GNU ]]; then
+    if [[ $IMGCAT_BASE64_VERSION =~ GNU ]]; then
```

Comment:
If `IMGCAT_BASE64_VERSION` contains a space, will it be a problem without quotes?

Reply:
Sorry I kept you waiting. Unlike `[`, `[[` does not allow word splitting of arguments (it takes the result of expansion as a single argument).

```bash
str="with space"

[ $str = $str ] # fails with:  bash: [: too many arguments

[[ $str = $str ]] # works as intended
```